app/services/availability_service.py

```
@@ -30,87 +31,172 @@
+                if not isinstance(item, dict):
+                    continue
+                if item.get("day_type") != day_type:
+                    continue
+                amount = item.get("amount")
+                if isinstance(amount, (int, float)) and amount > 0:
+                    slot_price += int(amount)
+
+        return max(slot_price, 0)
+
+    def calculate_total_price(self, room_detail, date_str: str, hour_slots: List[str]) -> int:
+        """?붿껌 ?щ’ ?꾩껜??珥?媛寃?怨꾩궛."""
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF002)